tools/import/import.css

```diff
@@ -123,3 +123,17 @@
   background-color: var(--gray-25);
 }
 
+.spinner {
```

Comment:
Candidate for `styles.css`?

Reply:
I'm not sure.  Wouldn't I have to make it more generic for everyone (flexible height, width, border width, margins, colors variants, etc.).